src/backend/distributed/transaction/transaction_management.c

```diff
@@ -621,6 +627,11 @@ CoordinatedSubTransactionCallback(SubXactEvent event, SubTransactionId subId,
 			}
 			PopSubXact(subId);
 
+			/*
+			 * Clear Cache Table
+			 */
+			InvalidateMetadataSystemCache();
```

Comment:
some workloads may have frequent subtransaction aborts, or even regular aborts

This would make all of those workloads quite expensive.

Can we do something more specific to the CREATE EXTENSION issue?